docs/source/json-api/production-setup.rst

```diff
@@ -0,0 +1,300 @@
+or rotating files) this information will be part of the log description.
+Using a logging target compatible with the Logstash Logback Encoder allows to have rich
+logs with structured information about the event being logged.
+
+
+
+Metrics
+*******
+
+Enable and configure reporting
+------------------------------
+
+
+To enable metrics and configure reporting, you can use the two following CLI options:
+
+- ``--metrics-reporter``: passing a legal value will enable reporting; the accepted values
+  are as follows:
+
+  - ``console``: prints captured metrics on the standard output
+
+  - ``csv://</path/to/metrics.csv>``: saves the captured metrics in CSV format at the specified location
+
+  - ``graphite://<server_host>[:<server_port>]``: sends captured metrics to a Graphite server. If the port
+    is omitted, the default value ``2003`` will be used.
+
+  - ``prometheus://<server_host>[:<server_port>]``: renders captured metrics
+    on a http endpoint in accordance with the prometheus protocol. If the port
+    is omitted, the default value ``55001`` will be used. The metrics will be
+    available under the address ``http://<server_host>:<server_port>/metrics``.
+
+- ``--metrics-reporting-interval``: metrics are pre-aggregated on the sandbox and sent to
+  the reporter, this option allows the user to set the interval. The formats accepted are based
+  on the ISO-8601 duration format ``PnDTnHnMn.nS`` with days considered to be exactly 24 hours.
+  The default interval is 10 seconds.
+
+Types of metrics
+================
+
+This is a list of type of metrics with all data points recorded for each.
+Use this as a reference when reading the list of metrics.
+
+Counter
+-------
+
+Number of occurrences of some event.
+
+Meter
+-----
+
+A meter tracks the number of times a given event occurred (throughput). The following data
+points are kept and reported by any meter.
+
+- ``<metric.qualified.name>.count``: number of registered data points overall
+- ``<metric.qualified.name>.m1_rate``: number of registered data points per minute
+- ``<metric.qualified.name>.m5_rate``: number of registered data points every 5 minutes
+- ``<metric.qualified.name>.m15_rate``: number of registered data points every 15 minutes
+- ``<metric.qualified.name>.mean_rate``: mean number of registered data points
+
+Timers
+------
+
+A timer records all metrics registered by a meter and by an histogram, where
+the histogram records the time necessary to execute a given operation (unless
+otherwise specified, the precision is nanoseconds and the unit of measurement
+is milliseconds).
+
+List of metrics
+===============
+
+The following is an exhaustive list of selected metrics
+that can be particularly important to track.
+
+``daml.http_json_api.command_submission_timing``
+------------------------------------------------
+
+A timer. Meters how long processing of a command submission request takes
```

Comment:
I would remove all of those `Meters` and just say "How long ...". More importantly, I think this should mention the time unit for each metric (even if it is always the same one).

Reply:
Regarding time unit for the metric , the Timer description earlier states this
`(unless
otherwise specified, the precision is nanoseconds and the unit of measurement
is milliseconds).`

The wordings for this were from the earlier documentation and I don't personally like either version
with the `Meters` or without it . If we want I can instead add 
`Measures latency for processing a command submission request`